modules/minimp3/resource_importer_mp3.cpp

```diff
@@ -87,42 +87,42 @@ bool ResourceImporterMP3::has_advanced_options() const {
 	return true;
 }
 void ResourceImporterMP3::show_advanced_options(const String &p_path) {
-	Ref<AudioStreamMP3> mp3_stream = import_mp3(p_path);
+	Ref<AudioStreamMP3> mp3_stream = load_from_file(p_path);
 	if (mp3_stream.is_valid()) {
 		AudioStreamImportSettingsDialog::get_singleton()->edit(p_path, "mp3", mp3_stream);
 	}
 }
 #endif
 
-Ref<AudioStreamMP3> ResourceImporterMP3::import_mp3(const String &p_path) {
-	Ref<FileAccess> f = FileAccess::open(p_path, FileAccess::READ);
-	ERR_FAIL_COND_V(f.is_null(), Ref<AudioStreamMP3>());
-
-	uint64_t len = f->get_length();
-
-	Vector<uint8_t> data;
-	data.resize(len);
-	uint8_t *w = data.ptrw();
-
-	f->get_buffer(w, len);
+void ResourceImporterMP3::_bind_methods() {
+	ClassDB::bind_static_method("ResourceImporterMP3", D_METHOD("load_from_buffer", "buffer"), &ResourceImporterMP3::load_from_buffer);
+	ClassDB::bind_static_method("ResourceImporterMP3", D_METHOD("load_from_file", "path"), &ResourceImporterMP3::load_from_file);
+}
```

Comment:
Usually, we put bind methods just before the constructor/destructor at the end of the file.